Sources/TensorFlow/Layers/Convolutional.swift

```diff
@@ -137,6 +142,8 @@ public struct Conv2D<Scalar: TensorFlowFloatingPoint>: Layer {
     @noDerivative public let padding: Padding
     /// The dilation factor for spatial dimensions.
     @noDerivative public let dilations: (Int, Int)
+    /// Workaround optionals not being handled by AD
```

Comment:
TF-1004 tracks enum differentiation support.
Could you please add references to TF-1004 in comments explaining that `useBias` is a workaround?
```suggestion
    // Note: `useBias` is a workaround for TF-1004: enum differentiation support.
```

Reply:
Does enum differentiation imply optional differentiation? (Sorry if this is a dumb question, just want to make sure.)

Reply:
Good point - enum differentiation support doesn't strictly imply `Optional` differentiation support.

I created a dedicated issue TF-1153 to track differentiation support for `Optional` operations, please reference that instead!

Reply:
Done, let me know if the phrasing is okay.

Reply:
Wording looks good!